index.html

```diff
@@ -6,176 +6,180 @@
         </div>
+        <section class="about">
+            <div class="about-content1">
+                <h1>About Myself</h1>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                <p class="connect">LET'S CONNECT</p>
+                <ul class="social-1">
+                    <li><a href="index.html"><img src="image/1.png" alt="social1"/></a></li>
+                    <li><a href="index.html"><img src="image/2.png" alt="social2"/></a></li>
+                    <li><a href="index.html"><img src="image/3.png" alt="social3"/></a></li>
+                    <li><a href="index.html"><img src="image/4.png" alt="social4"/></a></li>
+                    <li><a href="index.html"><img src="image/5.png" alt="social5"/></a></li>
+                </ul>
+                <button type="submit" class="project">Get my resume</button>
+            </div>
+            <div class="about-content2">
+                <ul class="skills">
+                    <li id="list"><a href="index.html">Languages</a></li>
+                    <div class="extra">
+                            <a href="index.html">
+                                <img src="image/Ellipse-1.png" alt="social1" />
+                                <p class="list-4">JavaScript</p>
+                            </a>
+                            <a href="index.html">
+                                <img src="image/Ellipse-3.png" alt="social1" />
+                                <p>HTML</p>
+                            </a>
+                            <a href="index.html">
+                                <img src="image/Ellipse-2.png" alt="social1" />
+                                <p>css</p>
+                            </a>
+                        </li>
+                    </div>
+                    <li id="list">Frameworks</li>
+                    <li id="list">Skills</li>
+                </ul>
+            </div>
+        </section>
+        <section class="contact-me">
+            <h1>Contact me</h1>
+            <p>
+                If you have an application you are interested in developing, 
+                a feature that you need built or a project that needs coding. 
+                i'd love to help with it.</p>
+            <form action="https://formspree.io/f/{form_id}" methop="post" class="text-me">
+                <input type="text" class="simple" placeholder="User name" maxlength="30" required>
+                <input type="email" class="simple" placeholder="Email" required>
+                <textarea class="text-area" placeholder="comment" required></textarea>
+                <button type="submit" class="project">Get in touch</button>
+            </form>       
+        </section>
```

Comment:
- Please I recommend, you try fixing the background image for the `contact me section`  as it is missing some of the background images from the Figma design.
- Please try to increase the width of the contact form fields as it is looking too narrow on desktop view. so it has a good UI and looks like that of the Figma design. 

##### Figma design 
![Screenshot from 2022-06-10 08-26-48](https://user-images.githubusercontent.com/58771507/173013910-7ebd6fd6-5450-40cc-9d9b-dff7af6eeee2.png)

##### Your design 
![Screenshot from 2022-06-10 08-26-19](https://user-images.githubusercontent.com/58771507/173013943-1e380be9-be2e-45ad-af6f-5272b8458d6b.png)